src/content_script.tsx

```diff
@@ -118,8 +118,9 @@ chrome.runtime.onMessage.addListener(function (
   }
 
   const verifiedTextInput = inputs.find((textInput) => verify(textInput));
-  if (verifiedTextInput != null) {
-    sendResponse({ success: true, address: verifiedTextInput.value });
+  // verifiedTextInput returns as {} when find fails for some reason.
+  if (Object.keys(verifiedTextInput as Object).length > 0)  {
```

Comment:
So `verifiedTextInput` is never null? Object.keys(null) would throw an exception so maybe safest to do a check like this?
```
verifiedTextInput != null && Object.keys(verifiedTextInput as Object).length > 0
```